[2024-08-29T03:46:11.547+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T03:46:11.567+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: country_data_pipeline.ingest_data manual__2024-08-29T03:46:06.861900+00:00 [queued]>
[2024-08-29T03:46:11.582+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: country_data_pipeline.ingest_data manual__2024-08-29T03:46:06.861900+00:00 [queued]>
[2024-08-29T03:46:11.583+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-29T03:46:11.632+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): ingest_data> on 2024-08-29 03:46:06.861900+00:00
[2024-08-29T03:46:11.640+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=845) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-29T03:46:11.641+0000] {standard_task_runner.py:72} INFO - Started process 854 to run task
[2024-08-29T03:46:11.640+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'country_data_pipeline', 'ingest_data', 'manual__2024-08-29T03:46:06.861900+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dags.py', '--cfg-path', '/tmp/tmpa38sssqr']
[2024-08-29T03:46:11.641+0000] {standard_task_runner.py:105} INFO - Job 183: Subtask ingest_data
[2024-08-29T03:46:11.749+0000] {task_command.py:467} INFO - Running <TaskInstance: country_data_pipeline.ingest_data manual__2024-08-29T03:46:06.861900+00:00 [running]> on host c63ae693b587
[2024-08-29T03:46:11.934+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='country_data_pipeline' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-29T03:46:06.861900+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-29T03:46:06.861900+00:00'
[2024-08-29T03:46:11.936+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T03:46:14.117+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-29T03:46:14.129+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T03:46:14.130+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=country_data_pipeline, task_id=ingest_data, run_id=manual__2024-08-29T03:46:06.861900+00:00, execution_date=20240829T034606, start_date=20240829T034611, end_date=20240829T034614
[2024-08-29T03:46:14.189+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-29T03:46:14.225+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-29T03:46:14.240+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
